src/utils/logger.py

```
@@ -26,76 +25,61 @@ def default(self, obj: Any) -> Any:
 
 
 def setup_logger(
-    name: str, log_file: Optional[pathlib.Path] = None, level: int = 20
-) -> Any:
+    log_file: Optional[pathlib.Path] = None, level: str = "INFO"
+) -> None:
     """配置并返回loguru日志记录器。
 
     Args:
         name (str): 日志记录器的名称（用于上下文标识）。
```

Ruff (0.14.2)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)